### PR TITLE
feat: Added 'Secret Exports' schema for Bicep

### PR DESCRIPTION
**Pull request by AlexanderSehr**
# Overview/Summary

- Added 'Secret Exports' schema for Bicep in dedicated interface page
- Updated order of bicep specs to match the way they're implemented in the modules (e.g., definitions last)

image

### CODE REVIEW

File: docs/content/specs-defs/specs/bicep/interfaces.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: Interfaces
+geekdocNav: true
+geekdocAlign: left
+geekdocAnchor: true
+url: /specs/bicep/interfaces/
+---
+
+Below are the interfaces/schemas for AVM Resource Modules features/extension resources only relevant to Bicep.
```
Comment (matebarabas) [inline docs/content/specs-defs/specs/bicep/interfaces.md:9]:
```suggestion
This page details the Bicep specific interfaces/schemas for AVM Resource Modules features/extension resources.
```

**Review by matebarabas — COMMENTED** [review]
Please check/incorporate my recommendations. Thanks!

File: docs/content/specs-defs/specs/bicep/interfaces.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+{{< toc >}}
+
+<br>
+
+## Secrets export
+
+The export of secrets from inside a module to a provided Key Vault reference is something that can be implement as per demand following the below schema.
+It's implementation provides a secure way around Bicep's current limitation to be unable to securely output secrets.
```
[the remark this answers is not in the document]
Comment (AlexanderSehr) [reply docs/content/specs-defs/specs/bicep/interfaces.md:18]: How about
> _This implementation provides a secure way around the current limitation of Bicep, which is to provide a secure template output_

File: docs/content/specs-defs/specs/bicep/interfaces.md (hunk 1 above)
Comment (AlexanderSehr) [reply docs/content/specs-defs/specs/bicep/interfaces.md:9]: I guess it should be `Bicep-specific`

File: docs/content/specs-defs/specs/bicep/interfaces.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (matebarabas) [reply docs/content/specs-defs/specs/bicep/interfaces.md:18]: I changed it to something very close :)

**Comment by AlexanderSehr** [comment]
All comments addressed. Thanks @matebarabas

**Review by matebarabas — APPROVED** [review]
With my comments addressed, the content's "language part" looks good to me. Please check the code aspect with the technical folks as necessary. Thanks!

**Pull request merged**